ui/src/views/Profiles.vue

```diff
@@ -213,6 +219,17 @@ export default defineComponent({
       }
       return port;
     },
+    firstFreeSeed(): number {
+      let seed = 100000000;
+      while (
+        this.profiles.find(
+          (profile) => profile.options["datashield.seed"] == seed
```

Comment:
In javascript it's `===` instead of `==` (don't ask me why 🙈)

Reply:
There was a reason behind this, VSCode didn't like the comparison between "string" and "number"... Changing the `Ref<Profile[]>` `Profile.options["datashield.seed"]` to `number` AND `Profile.datashieldSeed` to `number` also fixes this, with the use of `===`.

Reply:
After speaking with Dick: his understanding is that at some point we want to support 64 bit integers and strings in this `options.datashield.seed` field, so that's why I would prefer to keep it `==` so that in the future when we do support strings this part doesn't clash.